src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -1534,6 +1564,209 @@ subroutine init_atm_static(mesh, dims, configs)
 
  end subroutine init_atm_static
 
+
+ !***********************************************************************
+ !
+ !  routine init_atm_map_static_data
+ !
+ !> \brief   Map values of static datasets to cells
+ !> \author  Miles Curry
+ !> \date    25 January 2020
+ !> \details
+ !>  Given a initialized geotile manager object, an initialized KD tree of cell centers
+ !> (xCell, yCell, zCell), and two function pointers, this subroutine maps pixels of
+ !> a Geogrid binary format dataset (as described in the WRF User's Guide 3-41) to the
+ !> cells they reside in. The geogrid binary format is described in the WRF User's Guide
```

Comment:
The reference to the WRF User's Guide 3-41 is repeated from the previous sentence. Since page numbers to change from one version of the  User's Guide to another, we might be safer simply referencing Chapter 3.